templates/page.values.php

```diff
@@ -50,5 +50,8 @@
 		</ul>
 
 		<p class="details-link"><a href="<?php p($theme->getBaseUrl()); ?>" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', $theme->getName())); ?></a></p>
+		<div style="text-align: center;">
+			<h3><?php p($l->t('This Nextcloud is on version') . ' ' . \OC_Util::getVersionString()); ?></h3>
```

Comment:
Probably better this way to make sure the version is properly embedded in translated text:

```suggestion
			<h3><?php p($l->t('This Nextcloud is on version %s', [ \OC_Util::getVersionString() ])); ?></h3>
```

Reply:
done